### PR TITLE
[Docs] Features + release notes

### PR DESCRIPTION
**Pull request by brimoor**
## Release Notes

- **New Features**
  - Introduced a **Data Quality panel** for scanning datasets for quality issues.
  - Added a **Model Evaluation panel** for visualizing model evaluation results.
  - Enhanced **Query Performance** features for optimizing large dataset queries.
  - Introduced a **Data Lens** for exploring and importing samples from external sources.

- **Documentation Updates**
  - Expanded documentation on **Data Quality**, **Model Evaluation**, and **Query Performance**.
  - Improved clarity and organization across various sections, including FiftyOne Teams and the FiftyOne App.
  - Added comprehensive guides for developing plugins, including design overview and examples.

- **Bug Fixes**
  - Corrected release dates and formatting in release notes for previous versions.

### CODE REVIEW

**Review by minhtuev — DISMISSED** [review]
Query Performance change looks good, thanks Brian! ✅

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -3,17 +3,140 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+FiftyOne Teams 2.2.0
+--------------------
+*Released December 3, 2024*
+
+Includes all updates from :ref:`FiftyOne 1.1.0 <release-notes-v1.1.0>`, plus:
```
Comment (ritch) [inline docs/source/release-notes.rst:10]: Should there be a note about invitations and/or any other non-samples related changes?

File: docs/source/release-notes.rst [hunk 2]
```diff
@@ -3,17 +3,140 @@ FiftyOne Release Notes
...
+  :func:`compute_visualization() <fiftyone.brain.compute_visualization>`,
+  :func:`compute_uniqueness() <fiftyone.brain.compute_uniqueness>`, and
+  :func:`compute_representativeness() <fiftyone.brain.compute_representativeness>`
+  `#201 <https://github.com/voxel51/fiftyone-brain/pull/201>`_,
+  `#204 <https://github.com/voxel51/fiftyone-brain/pull/204>`_
+- Upgraded the :ref:`Pinecone integration <pinecone-integration>` to support
+  `pinecone-client>=3.2`
+  `#202 <https://github.com/voxel51/fiftyone-brain/pull/202>`_
+
+Plugins
```
Comment (ritch) [inline docs/source/release-notes.rst:90]: Added [skip_prompt](https://github.com/voxel51/fiftyone/blob/a6b5481a6f5627ac201fa30bbaa8126bda5cb2d3/fiftyone/operators/executor.py#L757) - that should be in here.

File: docs/source/teams/query_performance.rst [hunk 3]
```diff
@@ -1,147 +1,200 @@
...
+Deleting indexes/summaries
 --------------------------
 
-.. warning::
+You can delete a custom index or summary field by clicking its trash can icon
+in the actions column of the panel.
+
+.. _query-performance-disable:
+
+Disabling query performance
```
Comment (minhtuev) [inline docs/source/teams/query_performance.rst:163]: Small nit: `Query Performance`

File: docs/source/teams/query_performance.rst [hunk 4]
```diff
@@ -1,147 +1,200 @@
...
+
+Query performance is enabled by default for all datasets. This is generally the
+recommended setting for all large datasets to ensure that queries are
+performant.
+
+However, in certain circumstances you may prefer to disable query performance,
+which enables the App's sidebar to show additional information such as
+label/value counts that are useful but more expensive to compute.
+
+You can enable/disable query performance for a particular dataset for its
```
Comment (minhtuev) [inline docs/source/teams/query_performance.rst:174]:
```suggestion
You can enable/disable Query Performance for a particular dataset for its
```

File: docs/source/teams/query_performance.rst [hunk 5]
```diff
@@ -1,147 +1,200 @@
...
+lifetime (in your current browser) via the gear icon in the Samples panel's
+actions row:
+
+.. image:: /images/app/app-query-performance-disabled.gif
+    :alt: app-query-performance-disabled
+    :align: center
+
+You can also enable/disable query performance via the status button in the
```
Comment (minhtuev) [inline docs/source/teams/query_performance.rst:182]:
```suggestion
You can also enable/disable Query Performance via the status button in the
```

File: docs/source/user_guide/app.rst [hunk 6]
```diff
@@ -554,6 +542,30 @@ field:
 
     Numeric field filters are not supported by wildcard indexes.
 
+.. _app-disasbling-query-performance:
+
+Disabling query performance
```
Comment (minhtuev) [inline docs/source/user_guide/app.rst:547]:
```suggestion
Disabling Query Performance
```

File: docs/source/user_guide/app.rst [hunk 7]
```diff
@@ -554,6 +542,30 @@ field:
...
+---------------------------
+
+Query performance is enabled by default for all datasets. This is generally the
+recommended setting for all large datasets to ensure that queries are
+performant.
+
+However, in certain circumstances you may prefer to disable query performance,
```
Comment (minhtuev) [inline docs/source/user_guide/app.rst:554]:
```suggestion
However, in certain circumstances you may prefer to disable Query Performance,
```

File: docs/source/release-notes.rst (hunk 1 above)
Comment (brimoor) [reply docs/source/release-notes.rst:10]: Ah yes, added a note for SMTP configuration 👍

File: docs/source/release-notes.rst (hunk 2 above)
Comment (brimoor) [reply docs/source/release-notes.rst:93]: Added thanks 👍

File: docs/source/teams/query_performance.rst (hunk 3 above)
Comment (brimoor) [reply docs/source/teams/query_performance.rst]: Updated all to `Query Performance` 👍

**Review by ritch — APPROVED** [review]

**Review by minhtuev — APPROVED** [review]
LGTM 🚀

File: docs/source/teams/data_quality.rst [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+Samples tab.
+
+The panel's home page shows a list of the available issue types and their
+current analysis/review status:
+
+-   **Brightness**: scans for images that are unusually bright or dim
+-   **Blurriness**: scans for images that are abnormally blurry or sharp
+-   **Aspect Ratio**: scans for images that have extreme aspect ratios
+-   **Entropy**: scans for images that have unusually small or large entropy
+-   **Near Duplicates**: leverages embeddings to scan for
```
Comment (jleven) [inline docs/source/teams/data_quality.rst:30]: coming soon!

File: docs/source/teams/data_quality.rst [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+
+-   **Brightness**: the brightness of each image is stored in a `brightness`
+    field of the sample
+-   **Blurriness**: the blurriness of each image is stored in a `blurriness`
+    field of the sample
+-   **Aspect Ratio**: the aspect ratio of each image is stored in an
+    `aspect_ratio` field of the sample
+-   **Entropy**: the entropy of each image is stored in an `entropy` field of
+    the sample
+-   **Near Duplicates**: the nearest neighbor distance of each sample is stored
```
Comment (jleven) [inline docs/source/teams/data_quality.rst:108]: coming soon!

**Pull request merged**